web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -126,24 +174,58 @@ const TeamReviews = ({ periodId }) => {
     [setNewRequestOpen]
   );
 
+  const reviewAssignmentsUrl = '/services/review-assignments';
+
   useEffect(() => {
-    if (currentMembers && currentMembers.length > 0) {
-      isAdmin && includeAll
-        ? setTeamMembers(
-            currentMembers.filter(member => member?.id !== currentUser?.id)
-          )
-        : includeAll
-          ? setTeamMembers(subordinates)
-          : setTeamMembers(myTeam);
+    loadTeamMembers();
```

Comment:
This uses a new endpoint to get all the review assignments for a given review period.